gensim/scripts/make_wikicorpus.py

```diff
@@ -62,6 +62,12 @@
         print(globals()['__doc__'] % locals())
         sys.exit(1)
     inp, outp = sys.argv[1:3]
+    print "Checking if output directory exists: "+'/'.join(outp.split("/")[:-1])
```

Comment:
No need for this `print` -- just raise the exception.
Also, we'd be using `logger`, not printing.